### PR TITLE
GEODE-6861: separate static config object from runtime info in the Cl…

### PR DESCRIPTION
**Pull request by jinmeiliao**
…usterManagementResult

Co-authored-by: Joris Melchior <joris.melchior@gmail.com>

* use ConfigRealizer to retrieve the runtime information for configuration on each member
* list result would return a List of Response object which has a configuration object and a list runtime objects

Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

**Comment by jinmeiliao** [comment]
@Petahhh @jackw26

File: geode-management/src/main/java/org/apache/geode/management/runtime/RuntimeInfo.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ */
+
+package org.apache.geode.management.runtime;
+
+import java.io.Serializable;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, property = "class")
+public interface RuntimeInfo extends Serializable {
```
Comment (onichols-pivotal) [inline geode-management/src/main/java/org/apache/geode/management/runtime/RuntimeInfo.java]: Could `extends Serializable` be dropped?  Jackson does not require it; however adding it increases the attack surface for Java serialization vulnerability exploits.

Comment (jinmeiliao) [reply geode-management/src/main/java/org/apache/geode/management/runtime/RuntimeInfo.java]: The implementation of RuntimeInfo (like RuntimeRegionInfo) is gathered on each member and will need to be transferred back to the locator via function calls, so it needs to be Serializable. Geode protects itself from Java serialization vulnerability by allowing only those declared in "sanctioned-serializable.txt" being sent across the wires. If we don't declare it to be Serializable, the function execution will fail.

Comment (onichols-pivotal) [reply geode-management/src/main/java/org/apache/geode/management/runtime/RuntimeInfo.java]: Ok, that makes sense.

**Pull request merged**